lua/cfc_err_forwarder/init.lua

```diff
@@ -49,10 +49,6 @@ include( "cfc_err_forwarder/branch.lua" )
 local Discord = EF.Discord
 local Forwarder = EF.Forwarder
 
-cvars.AddChangeCallback( Config.groomInterval:GetName(), function( _, _, new )
```

Comment:
I use this feature a lot in development, would like to maintain that functionality